modules/doc/content/getting_started/examples_and_tutorials/tutorial01_app_development/step07_parallel.md

```diff
@@ -1,6 +1,107 @@
+|   Steady::PicardSolve                    |     1 |      0.000 |      0.000 |   0.04 |       0.065 |      0.065 |  51.27 |      0.065 |      0.065 |  51.31 |
+|     FEProblem::solve                     |     1 |      0.007 |      0.007 |   5.63 |       0.058 |      0.058 |  45.47 |      0.065 |      0.065 |  51.10 |
+|       FEProblem::computeResidualInternal |     4 |      0.000 |      0.000 |   0.00 |       0.018 |      0.004 |  14.10 |      0.018 |      0.004 |  14.11 |
+|       FEProblem::computeJacobianInternal |     2 |      0.000 |      0.000 |   0.00 |       0.040 |      0.020 |  31.25 |      0.040 |      0.020 |  31.26 |
+|     FEProblem::outputStep                |     1 |      0.000 |      0.000 |   0.08 |       0.000 |      0.000 |   0.00 |      0.000 |      0.000 |   0.08 |
+|   Steady::final                          |     1 |      0.000 |      0.000 |   0.01 |       0.000 |      0.000 |   0.03 |      0.000 |      0.000 |   0.04 |
+|     FEProblem::outputStep                |     1 |      0.000 |      0.000 |   0.02 |       0.000 |      0.000 |   0.01 |      0.000 |      0.000 |   0.03 |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------
+```
+
+The report indicates that the total duration of the execution was approximately 0.127 seconds (obviously this will vary depending on hardware).
+
+To test the parallel scalability of this [!ac](FE) model without modifying the input file, the `-r` command-line option can be used to automatically perform a uniform mesh refinement before solving:
+
+```bash
+cd ~/projects/babbler
+./babbler-opt -i problems/pressure_diffusion.i -r 1 # splits QUAD elements into four smaller QUADs
+```
+
+Now, the initial report should show that there are 4221 [!ac](DOFs)-per-process and the performance report might indicate that the execution took roughly four times as long. Since there are about four times as many DOFs on a single process for the refined mesh as there were for the unrefined, the problem ought to be solved with four processes to obtain a similar level of performance. Thus, run the input file using the following commands:
+
+```bash
+cd ~/projects/babbler
+mpiexec -n 4 ./babbler-opt -i problems/pressure_diffusion.i -r 1 # run refined mesh on four procs
+```
+
+Now the printout should indicate that there are 1077 [!ac](DOFs)-per-process, roughly equal to that of the unrefined mesh ran in serial. The total execution time will be shorter than that of the refined mesh in serial, but certainly not as short as the unrefined in serial. While this might suggest poor scaling, one may now realize that this is why an ideal number of [!ac](DOFs)-per-process (20,000) exists at all. There is simply not enough calculations to perform to for the problem to scale well in parallel. In practice, a single process is sufficient for any MOOSE [!ac](FE) problem that has less than 20,000 total [!ac](DOFs).
```

Comment:
```suggestion
Now the printout should indicate that there are 1077 [!ac](DOFs)-per-process, roughly equal to that of the unrefined mesh ran in serial. The total execution time will be shorter than that of the refined mesh in serial, but certainly not as short as the unrefined in serial. While this might suggest poor scaling, one may now realize that this is why an ideal number of [!ac](DOFs)-per-process (20,000) exists at all. There is simply not enough calculations to perform for the problem to scale well in parallel. In practice, a single process is sufficient for any MOOSE [!ac](FE) problem that has less than 20,000 total [!ac](DOFs).
```